src/DevBetterWeb.Web/Program.cs

```diff
@@ -20,7 +20,7 @@ public static async Task Main(string[] args)
         {
             var builder = CreateHostBuilder(args);
 
-            //string env = "Development";
+            string env = "Development";
```

Comment:
This probably needs to be the way it was. We don't want to force the environment to always be Development when we deploy to production.